ejb3/src/main/java/org/jboss/as/ejb3/subsystem/EJB3Subsystem20Parser.java

```diff
@@ -132,6 +134,14 @@ protected void parseStatefulBean(final XMLExtendedStreamReader reader, final Lis
             }
             // found the mandatory attribute
             missingRequiredAttributes.remove(attribute);
+
+            // differentiate between non-HA and HA profile
+            if (clusteredCacheRef == null) {
+                EJB3SubsystemRootResourceDefinition.DEFAULT_SFSB_CACHE.parseAndSetParameter(cacheRef, ejb3SubsystemAddOperation, reader);
```

Comment:
Suggestion from Brian: we can throw in a friendly INFO that we are doing some sort of mapping here.